recipes/trexio/meta.yaml

```diff
@@ -0,0 +1,58 @@
+
+{% set name = "trexio" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/TREX-CoE/trexio/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 232866c943b98fa8a42d34b55e940f7501634eb5bd426555ba970f5c09775e83
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - autoconf  
+    - automake  
+    - libtool   
+    - make      
+    - gnuconfig 
+    - pkg-config
+
+  host:
+    - hdf5
+    # the dependencies below are not required by TREXIO but they get exported from HDF5
+    # and if they are not listed here than the `conda-build --error-overlinking` fails
+    - zlib    
+    - openssl 
+    - libcurl 
+
+  run:
+    - hdf5 
```

Comment:
This shouldn't be required because hdf5 has a run export for itself.

Reply:
Done. 

Here I had one question: since `trexio` depends on `hdf5`, should this dependency be somehow exported as a runtime dependency? Or this is taken into account by `conda` when listing `hdf5` under `requirements/host`?

Reply:
Because the recipe for hdf5 contains a run export, conda-build automatically adds hdf5 to the `requirements/run` section of your recipe when you add it to the `requirements/host` section.